### PR TITLE
[Release 1.1] Release notes for Hotfix v1.1.4

### PR DESCRIPTION
**Pull request by cheenamalhotra**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Stable Release 1.1.4] - 2021-03-09
+
+### Fixed
+- Fixed wrong results issues by changing the timeout timer to ensure a correct execution state [#950](https://github.com/dotnet/SqlClient/pull/950)
+- Fixed MARS header contains errors issue against .NET Framework 4.8+ [#959](https://github.com/dotnet/SqlClient/pull/959)
+
+
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:13]: Extra line.

File: release-notes/1.1/1.1.4.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## General Availability of Microsoft.Data.SqlClient 1.1.4 released 9 March 2021
+
+This update brings the below changes over the previous release:
+
+### Fixed
+- Fixed wrong results issues by changing the timeout timer to ensure a correct execution state [#950](https://github.com/dotnet/SqlClient/pull/950)
+- Fixed MARS header contains errors issue against .NET Framework 4.8+ [#959](https://github.com/dotnet/SqlClient/pull/959)
+
+
```
Comment (DavoudEshtehari) [inline release-notes/1.1/1.1.4.md:11]: Extra line.

**Review by DavoudEshtehari — APPROVED** [review]

File: release-notes/1.1/1.1.4.md (hunk 2 above)
Comment (cheenamalhotra) [reply release-notes/1.1/1.1.4.md:11]: 2 lines are always added before next superior heading.
(You can compare with other files)

File: CHANGELOG.md (hunk 1 above)
Comment (cheenamalhotra) [reply CHANGELOG.md:13]: same applies here.

**Review by JRahnama — APPROVED** [review]

File: release-notes/1.1/1.1.4.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- Microsoft.Identity.Client 3.0.8
+
+### Always Encrypted with secure enclaves
+
+In general, existing documentation which uses System.Data.SqlClient on .NET Framework should now work with .NET Core, too.
+
+- [Develop using Always Encrypted with .NET Framework Data Provider](https://docs.microsoft.com/sql/relational-databases/security/encryption/develop-using-always-encrypted-with-net-framework-data-provider)
+- [Always Encrypted: Protect sensitive data and store encryption keys in the Windows certificate store](https://docs.microsoft.com/azure/sql-database/sql-database-always-encrypted)
+
+In order to use the enclave feature, connection strings should include the required attestation protocol and attestation URL. Examples:
```
Comment (DavoudEshtehari) [inline release-notes/1.1/1.1.4.md:58]: NIT: Break down this line from the `Examples` and remove the `s`.

File: release-notes/1.1/1.1.4.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- System.Diagnostics.DiagnosticSource 4.5.0
+- System.Memory 4.5.1
+- System.Security.Principal.Windows 4.5.0
+- System.Text.Encoding.CodePages 4.5.0
+- System.Configuration.ConfigurationManager 4.5.0
+- Microsoft.Identity.Client 3.0.8
+
+### Always Encrypted with secure enclaves
+
+In general, existing documentation which uses System.Data.SqlClient on .NET Framework should now work with .NET Core, too.
```
Comment (David-Engel) [inline release-notes/1.1/1.1.4.md:53]:
```suggestion
In general, existing documentation that uses System.Data.SqlClient on .NET Framework should now work with .NET Core, too.
```

File: release-notes/1.1/1.1.4.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+- [Develop using Always Encrypted with .NET Framework Data Provider](https://docs.microsoft.com/sql/relational-databases/security/encryption/develop-using-always-encrypted-with-net-framework-data-provider)
+- [Always Encrypted: Protect sensitive data and store encryption keys in the Windows certificate store](https://docs.microsoft.com/azure/sql-database/sql-database-always-encrypted)
+
+In order to use the enclave feature, connection string should include the required attestation protocol and attestation URL.
```
Comment (David-Engel) [inline release-notes/1.1/1.1.4.md:58]:
```suggestion
In order to use the enclave feature, the connection string should include the required attestation protocol and attestation URL.
```

**Review by David-Engel — APPROVED** [review]

**Pull request merged**